### ISSUE TITLE
update check points to http instead of https

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1979]
To check for the latest version the application checks a json file on github io called latestversion.json. however the link currently coded in `src/sas/sasview/local_config.py` is http://www.sasview.org/latestversion.json it should instead point to https://www.sasview.org/latestversion.json.  This is a minor issue since in most cases the address will just be redirected to the secure version. There may be occasions however where the client computer will refuse the redirect a problem that is likely to increase a security becomes a bigger and bigger concern.

**Comment by smk78** [comment]
Would it not be sensible to get this into 5.0.5?

**Comment by butlerpd** [comment]
I agree the case could certainly be made - and in fact was how I first assigned it.  However, I then remembered the rules of the road agreed to just recently and as a problem that has existed since the beginning (going back as far as 5.0.1 or before) it was not clear to me that it should be pushed to the release branch, particularly since it is unlikely to be causing a problem to anyone.  

That said the change is so trivial it should not pose any problem whatsoever to include it I would think.
